packages/common-all/src/fuse.ts

```diff
@@ -165,4 +168,40 @@ export class NoteLookupUtils {
     const nodes = _.map(childrenOfRoot, (ent) => notes[ent]).concat(roots);
     return nodes;
   };
+
+  static async lookup({
```

Comment:
can we call this something more descriptive?
I'm thinking something like `fetchNodesFromEngine`.

The regular `fetchPickerResults` should probably also be refactored to use this util in the future

Reply:
wouldn't use `fetchNodesFromEngine` because `queryNotes` does that already. this method wraps `fetchRootResults`, `queryNotes`, and takes into account the direct stub filter. this is why I ended up using `lookup` - in the future, we can imagine the web client and other clients of Dendron using this directly to do `lookup`